basyx.examples.timerseries/src/main/java/org/eclipse/basyx/examples/timeseries/TimeSeriesSubmodel/InfluxDBInvokables.java

```diff
@@ -0,0 +1,161 @@
+/*******************************************************************************
+ * Copyright (C) 2023 the Eclipse BaSyx Authors
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+ * LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ * 
+ * SPDX-License-Identifier: MIT
+ ******************************************************************************/
+
+package org.eclipse.basyx.examples.timeseries.TimeSeriesSubmodel;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+import org.eclipse.basyx.submodel.metamodel.api.qualifier.haskind.ModelingKind;
+import org.eclipse.basyx.submodel.metamodel.api.submodelelement.ISubmodelElement;
+import org.eclipse.basyx.submodel.metamodel.map.submodelelement.SubmodelElementCollection;
+import org.eclipse.basyx.submodel.metamodel.map.submodelelement.dataelement.property.Property;
+
+import org.eclipse.basyx.examples.timeseries.influxDB.QueryBuilder;
+import org.eclipse.basyx.examples.timeseries.influxDB.InfluxDBConnection;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/*
+ * Creates invokables for TimeSeries query operation
+ * 
+ * author Al-Obaidi
+ */
+
+public class InfluxDBInvokables {
+    final static String INFLUXDB_TOKEN = "d28e8bebb73e59c0cd88e8a1ac7855aa";
+    final static String INFLUXDB_BUCKET = "airsensors";
+    final static String INFLUXDB_ORG = "basyx";
+    final static String INFLUXDB_URL = "http://influxdb:8086";
+    final static String PIVOT_RULE = "rowKey: [\"_time\"], columnKey: [\"_field\"],valueColumn: \"_value\"";
+    final static String FILTER_RULE = "fn: (r) => r._measurement == \"%s\"";
```

Comment:
Reorder the modifiers to comply with the Java Language Specification. (static final instead of final static)
Lines 52 ~ 57